pkg/agent/main.go

```diff
@@ -204,7 +204,7 @@ func run() error {
 		if !isConnect() {
 			wsURL += "/register"
 		}
-		logrus.Infof("Connecting to %s with token %s", wsURL, token)
+		logrus.Infof("Connecting to %s with token %s, params %s", wsURL, headers[Token], headers[Params])
```

Comment:
@ukinau we should avoid logging any sensitive info. Are there any sensitive data in headers[Params]? Also on that note, instead of logging the entire token, could you change it to log only first/last few chars of it?

Reply:
>Are there any sensitive data in headers[Params]?
For the node-agent parameter, I don't think there is sensitive data. Actually each value already exists in log. So technically we can reproduce headers[Params] data with current logs.
They include following data. 
 - address
 - internal address
 - roles
 - labels
 - etcd flag
 - control flag
 - worker flag
 - requestedHostname
 - dockerInfo

For the cluster-agent parameter, there seems be some sensitive data. All of them are not printed in current logs. So it's impossible to reproduce with current log.
They include following
 - service account caCert
 - service account token
 - kubernetes service endpoint (passed from environment variable)

I think if we consider `service account caCert` and `service account token` as a sensitive data, we should not print `headers[Params]` in the case of cluster-agent.
But for the node-agent, it's okay to print `headers[Params]` in my opinion.

>Also on that note, instead of logging the entire token, could you change it to log only first/last few chars of it?
let me make sure it. Actually even before this change, entire token has been printed in log.
I actually changed variable to pass from `token` to `headers[Token]`. But both of variable refer to same value. So I thought we wanted to print entire token for some reason. 
Do you think printing entire token(ClusterRegistrationToken) was not good idea in the first place? and want me to change to log only first/last few chars of it?

Reply:
> I think if we consider `service account caCert` and `service account token` as a sensitive data, we should not print `headers[Params]` in the case of cluster-agent.

It is indeed a sensitive data that should not be printed.

> Do you think printing entire token(ClusterRegistrationToken) was not good idea in the first place? and want me to change to log only first/last few chars of it?

Yes, lets change it, it's not right to expose the whole value in the log